user/enterprise/high-availability.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Enterprise High Availability Mode
+layout: en_enterprise
+
+---
+
+Travis CI Enterprise typically runs as a single container instance communicating with one or multiple workers and generally, this setup is quite stable and maintainable. However, in some cases -- especially volume licensing or by corporate requirement -- you might need to install Travis CI in High Availability (HA) mode. If you're interested, or might be interested, in HA, please let [us](mailto:enterprise@travis-cicom?subject:HA%2FMode) know and we can discuss options and help you get started. 
```

Comment:
Do we want a separate section about contacting us since as it is currently HA mode requires us to enable that functionality and they then have to sync the license or do we want to have this turned on by default? I'm a bit hesitant on turning it on by default right now since even for us we really only have IBM using it and it would be good for us to know who else is by having them contact us rather than letting any customer use it if they want and potentially running into a bad experience without us knowing.

Reply:
Ahh, I see we say it in the prerequisites section. I wonder if we should say it on the HA page as well in case people configure their single instance, but then decide later to look into HA they might go directly to this page rather than the prerequisites one.

Reply:
let's do both 👍